modules/nickname/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,67 @@
+module.exports.run = async function (client, oldGuildMember, newGuildMember) {
+
+    if (!client.botReadyAt) return;
+    if (newGuildMember.guild.id !== client.guild.id) return;
+    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.highest.position === oldGuildMember.guild.me.roles.highest.position) return;
+
+    const roles = client.configurations['nickname']['config'];
+    const moduleModel = client.models['nickname']['User'];
+
+    let rolePrefix = "";
+    let hoistrole
+    if (newGuildMember.roles.hoist === null) {
+        hoistrole = null;
+    } else {
+        hoistrole = newGuildMember.roles.hoist.id;
+    }
+
+    for (const role of roles) {
+        if (role.roleID === hoistrole) {
+            rolePrefix = role.prefix;
+        }
+    }
+
+    let user = await moduleModel.findOne({
+        attributes: ['userID', 'nickname'],
+        raw: true,
```

Comment:
why are we doing this here? Usually leaving raw to false allows us to use sequelize's module interface and other features, while still allowing direct attribute access. (i don't really enjoy working with sequelize either anymore, prisma is much more fun)

Reply:
Because I just copied it from somewhere else I used the database and it just works and I haven't tried it in this situation without raw. Will change if it works 👍